rest/route/github.go

```diff
@@ -194,10 +196,21 @@ func (gh *githubHookApi) Run(ctx context.Context) gimlet.Responder {
 		}
 
 	case *github.PushEvent:
+		grip.Debug(message.Fields{
+			"source":     "github hook",
+			"msg_id":     gh.msgID,
+			"event":      gh.eventType,
+			"event_data": event,
+		})
+		if isTag(event.GetRef()) {
```

Comment:
Do you need to have this same check for when the repotracker is triggered manually?

Reply:
🤔 I'm not sure what you mean. This check specifically validates a field for github events, since github push events can be for pushed commits or pushed tags. Is there somewhere else this happens?

Reply:
Oops should have been more clear. This will create a version if you push a tag for a project that has webhooks enabled, but what about a project that is using the repotracker polling method?

Reply:
Oh interesting. I didn't know that was really a thing but it does look like the boat some of the drivers are in. This wasn't included in the scope and is maybe out of scope for this ticket, but I'll create a ticket to investigate.

Reply:
@john-m-liu — @ablack12 and I talked about this on Slack. I'm going to open a ticket to move everyone to git hooks. I don't think we should support repotracker-only projects. The user experience for those is substantially worse. It can be a requirement of this project that users are using webhooks.